.travis/downstream-environments.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+body='{
+    "request": {
+        "branch": "master",
+        "message": "Update dummy-api to '"$1"'"
+    }
+}'
+
+curl -s -X POST \
```

Comment:
Tend to prefer the long form when not typing (so I don't have to remember that it means `silent` etc).